src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -3933,13 +3935,15 @@ CreateTable CreateTable():
         ")"
         (     parameter=CreateParameter() { tableOptions.addAll(parameter); } )*
         )
+        // table properties
```

Comment:
If my understanding of the JavaCC grammar is correct, then this should match at least the row movement clause from here: https://docs.oracle.com/cd/B19306_01/server.102/b14200/img_text/table_properties.htm

For some reason, I'm not able to actually do a `enable row movement AS select()` parsing, I am thinking some of these productions are in the column_properties or the `K_AS` portion needs to be optional, or `*` instead?

Reply:
That's a reasonable way to start to first do small steps. 

Do you get an exception?

Reply:
yeah, I can try to pull up the exception with what happens with `enable row movement AS ...` , I think this at least satisfies the feature requested on the issue. I'm thinking that it would take a bit of mucking with the grammar to get `enable row movement AS select()` working. The way it is now the `enable row movement` part is optional and then it expects that `AS path`

Reply:
Here's what happens when I try to support AS after the enable part:

```
CREATE TABLE test (startdate DATE) DISABLE ROW MOVEMENT AS select 1 from dual
```

Doesn't seem that the AS token is recognized. That might be because it is in the optional part for `columns`? 
```
Call:	Statement
  Call:	SingleStatement
    Call:	CreateTable
      Consumed token: <"CREATE" at line 1 column 1>
      Consumed token: <"TABLE" at line 1 column 8>
      Call:	Table
        Call:	RelObjectNameList
          Call:	RelObjectNameExt
            Call:	RelObjectName
              Call:	RelObjectNameWithoutValue
                Consumed token: <<S_IDENTIFIER>: "test" at line 1 column 14>
              Return: RelObjectNameWithoutValue
            Return: RelObjectName
          Return: RelObjectNameExt
        Return: RelObjectNameList
      Return: Table
      Consumed token: <"(" at line 1 column 19>
      Call:	ColumnDefinition
        Call:	RelObjectName
          Call:	RelObjectNameWithoutValue
            Consumed token: <<S_IDENTIFIER>: "startdate" at line 1 column 20>
          Return: RelObjectNameWithoutValue
        Return: RelObjectName
        Call:	ColDataType
          Consumed token: <<K_DATETIMELITERAL>: "DATE" at line 1 column 30>
        Return: ColDataType
      Return: ColumnDefinition
      Consumed token: <")" at line 1 column 34>
      Call:	RowMovement
        Consumed token: <"DISABLE" at line 1 column 36>
        Consumed token: <"ROW" at line 1 column 44>
        Consumed token: <"MOVEMENT" at line 1 column 48>
      Return: RowMovement
    Return: CreateTable
  Return: SingleStatement
Return: Statement

net.sf.jsqlparser.JSQLParserException
	at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:65)
	at net.sf.jsqlparser.parser.CCJSqlParserUtil.parse(CCJSqlParserUtil.java:40)
	at net.sf.jsqlparser.statement.create.CreateTableTest.foo(CreateTableTest.java:641)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:230)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:58)
Caused by: net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "AS" "AS"
```

Reply:
Alright, I was able to get it working. I had to move my clauses to be outside the column properties portion (and create table) portion.

Reply:
Fixed with f6e7ff8!